docs/content/stable/yugabyte-platform/security/authentication/oidc-authentication-aad.md

```diff
@@ -185,13 +185,16 @@ For information on configuring flags in YugabyteDB Anywhere, refer to [Edit conf
 
 #### ysql_hba_conf_csv
 
-The `ysql_hba_conf_csv` flag must be set to support using JWTs for authentication. The parameters to include in the configuration file record are as follows:
+The `ysql_hba_conf_csv` flag must be set to support using JWTs for authentication. The parameters to include in the configuration file record are described in the following table:
 
-- `jwt_map` - the user-name map used to translate claim values to database roles. Optional if you aren't using the default Subject claim values.
-- `jwt_issuers` - the first part of the discovery URL (`login.microsoftonline.com/<tenant_id>/v2.0`)
-- `jwt_audiences` - the audience or target app for the token, which in this case is the client ID of the application you registered.
-- `jwt_matching_claim_key` - the email attribute you set (for example, `preferred_username`). Optional if you aren't using the default Subject claim values.
-- `jwt_jwks_path` - The JSON Web Key Set (JWKS) is a set of keys containing the public keys used to verify any JWT. These can be uploaded as entries in a single file. When configuring the flag in YugabyteDB Anywhere, click **Add JSON web key set (JWKS)** to upload the JWKS.
+| Parameter | Description |
+| :-------- | :---------- |
+| `jwt_map` | The user-name map used to translate claim values to database roles. Optional if you aren't using the default Subject claim values. |
```

Comment:
@Vars-07 do we provide an example of this map file anywhere? What does it look like